### PR TITLE
Add meeting notes from August

### PR DESCRIPTION
**Pull request by encukou**
----
:books: Documentation preview :books:: https://docs-community--85.org.readthedocs.build/en/85/

### CODE REVIEW

File: docs/monthly-meeting/2023-08.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- Marcus Sherman / `@betteridiot`
+- Joe Kaufeld / `@itsthejoker`
+
+
+## Reports and celebrations
+
+- [PR to keep "translated" attribute on translated nodes](https://github.com/sphinx-doc/sphinx/pull/11502) landed and released! :tada: 
+    - https://github.com/python/docsbuild-scripts/issues/148#issuecomment-1648836409 has the details on how it's exposed!
+
+- [Hugo] CPython sprint at EuroPython: we had one room for code and another for documentation, and merged some 140 PRs in total! Thanks to Daniele for helping out and all the excellent docs advice! (Photos: https://mastodon.social/@hugovk/110792652209141428)
```
Comment (humitos) [inline docs/monthly-meeting/2023-08.md:35]: Wow! 140 PRs! Just wow! Congrats and thanks for your work!

**Review by humitos — APPROVED** [review]

File: docs/monthly-meeting/2023-08.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+- [Hugo] Plausible trial update
+  - Public Dashboard [link expired]
+  - Would be too expensive to pay for hosted version
+  - Should be easy to host it ourselves on PSF infra
+  - Next step:
+    - Ask PSF/Ee if we want to self-host, or pay for the service (or get sponsored)
+    - Let the SC look at the dashboard
+
+- [Hugo] Dark theme deploy: 3.11 RM Pablo said fine to deploy. To deploy, need to merge https://github.com/python/docsbuild-scripts/pull/161 and/or pin in `requirements.txt`
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2023-08.md:51]: This doesn't seem to generate a link in the preview -- this should fix it:
```suggestion
- [Hugo] Dark theme deploy: 3.11 RM Pablo said fine to deploy. To deploy, need to merge [python/docsbuild-scripts#161](https://github.com/python/docsbuild-scripts/pull/161) and/or pin in `requirements.txt` 
```

File: docs/monthly-meeting/2023-08.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    - Forward plan: Merge Daniele's PR and open an issue for a followup to remove/update more outdated content
+
+
+## Follow-ups from last month(s)
+
+* Thoughts on MDN icons (see e.g. the ToC sidebar for [Document](https://developer.mozilla.org/en-US/docs/Web/API/Document)), as a compact way to flag API status in overviews?
+
+    * :wastebasket: = deprecated
+    * :warning: = non-standard/implementation-specific
+    * 🧪 = experimental/unstable
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2023-08.md:80]: The `:...:` don't render in the preview, this should work:
```suggestion
    * 🗑 (`:wastebasket:`) = deprecated
    * ⚠ (`:warning:`) = non-standard/implementation-specific
    * 🧪 (`:test_tube:`) = experimental/unstable
```

File: docs/monthly-meeting/2023-08.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  * Followup from [soft deprecation discussion](https://discuss.python.org/t/formalize-the-concept-of-soft-deprecation-dont-schedule-removal-in-pep-387-backwards-compatibility-policy/27957/73)
+  * MVP should be up by next meeting:
+    * A list of everything that's deprecated
+    * docs on the directive in the devguide
+
+* Thoughts on putting a translation progress notice in docs pages (for each language)? [python/docsbuild-scripts#148](https://github.com/python/docsbuild-scripts/issues/148)
+
+  * Sounds like a good idea
+  * Sphinx issue is open
+    * This should be done in Sphinx first - Manuel will look into it (https://github.com/sphinx-doc/sphinx/issues/11157)
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2023-08.md:92]:
```suggestion
    * This should be done in Sphinx first - Manuel will look into it ([sphinx-doc/sphinx#11157](https://github.com/sphinx-doc/sphinx/issues/11157))
```

**Review by hugovk — APPROVED** [review]
Thanks, let's merge this.

Would someone like to do the September notes?

**Comment by egeakman** [comment]
> Thanks, let's merge this.
> 
> Would someone like to do the September notes?

I can do the September notes 👍

**Pull request merged**